### PR TITLE
MRG: ICA indexing

### PR DESCRIPTION
**Pull request by larsoner**
Simplifies ICA by using `ICA000` as the first label.

cc @jona-sassenhagen can you take a look and see if you agree it makes things simpler? I think you did a lot of the viz code. I will post the relevant CircleCI link once it's done.

### CODE REVIEW

**Comment by larsoner** [comment]
When I view this, CircleCI has botched the CSS somehow (!?), but you can still see the plots:

https://5461-1301584-gh.circle-artifacts.com/0/tmp/circle-artifacts.roaYEKx/html/auto_tutorials/plot_artifacts_correction_ica.html

cc @agramfort @dengemann @stfnrpplngr see if this makes sense to you

File: mne/viz/topomap.py [hunk 1]
```diff
@@ -947,18 +947,16 @@ def plot_ica_components(ica, picks=None, ch_type=None, res=64,
     if merge_grads:
         from ..channels.layout import _merge_grad_data
     for ii, data_, ax in zip(picks, data, axes):
-        if ii in ica.exclude:
-            ax.set_title('IC #%03d' % ii, fontsize=12, color='gray')
-        else:
-            ax.set_title('IC #%03d' % ii, fontsize=12)
+        kwargs = dict(color='gray') if ii in ica.exclude else dict()
+        ax.set_title(ica._ica_names[ii], fontsize=12, **kwargs)
```
Comment (jona-sassenhagen) [inline mne/viz/topomap.py]: Maybe `color="gray" if ii in ica.exclude else "black"`

**Comment by jona-sassenhagen** [comment]
I didn't do much in that part of the code, but it looks sensible to me.

**Comment by larsoner** [comment]
Ahh you're right, memory failed me. It was @jaeilepp (who probably won't have time to look). Thanks for looking

**Comment by agramfort** [comment]
no idea about the CSS issue. I would try to flush the cache of circle and
rebuild

diff LGTM

**Comment by larsoner** [comment]
@mmagnuski you might also be interested in this one if you have time

**Comment by larsoner** [comment]
Okay CircleCI fixed itself on the next commit

https://5464-1301584-gh.circle-artifacts.com/0/tmp/circle-artifacts.6IKpEb9/html/auto_tutorials/plot_artifacts_correction_ica.html

**Comment by agramfort** [comment]
great !

**Comment by agramfort** [comment]
Thanks @larsoner

**Comment by mmagnuski** [comment]
👍

**Comment by stfnrpplngr** [comment]
Great!

Am 26. August 2017 01:29:18 MESZ schrieb Mikolaj Magnuski <notifications@github.com>:

**Comment by aweinstein** [comment]
@larsoner It seems that this PR breaks the labeling of `ica.plot_sources`. Instead of the components being labeled 'IC #000', they get the labels from the name of the electrodes: 'Fp1', etc.

You can see the behavior in the following image.

ica

I think the offending line might be this one:

https://github.com/mne-tools/mne-python/blob/aaaee116644f71af645ed95db05b2cbaacaa3dcd/mne/viz/ica.py#L757

**Comment by larsoner** [comment]
Do you want to make a PR to fix it?

**Pull request merged**